README.md

```diff
@@ -20,23 +20,42 @@ Source code annotation tool offers an UI to annotate source code and review thes
 
 ## Installation
 
-## Github OAuth tokens
+## Environment Variables
```

Comment:
`Installation` and `Environment Variables` have the same size of header. It looks a bit weird:
<img width="285" alt="screen shot 2018-02-26 at 18 44 48" src="https://user-images.githubusercontent.com/406916/36685943-28bb183e-1b25-11e8-8a85-b9cf3687c215.png">